bims/templates/map_page/map-templates.html

```diff
@@ -28,7 +39,8 @@
         </div>
         <div class="right-panel-header">
             <div class="pull-right">
-                <img class="close-icon close-panel header-panel-button" src="/static/img/close_icon.svg">
+                <img class="close-icon close-panel header-panel-button"
```

Comment:
Next time please don't reformat the code, it's hard to check which changes.

Reply:
Roger